app/page.tsx

```diff
@@ -57,7 +59,8 @@ function getBranchInfosFromExperiment(recipe: NimbusExperiment) : BranchInfo[] {
       case 'infobar':
         branchInfo.id = value.messages[0].id
         branchInfo.ctrDashboardLink = getDashboard(template, branchInfo.id)
-        break
+        branchInfo.previewLink = getPreviewLink(_substituteLocalizations(value.content, recipe.localizations?.[Object.keys(recipe.localizations)[0]]));
```

Comment:
This `[Object.keys(recipe.localizations)[0]]` thing is kinda gross but it means we correctly access the first locale inside the localization object. We'll probably want to add a dropdown component that allows us to choose a locale from the available ones, to pass to this function.

Reply:
How about adding this in as an XXX comment?

Reply:
Will do :)

Reply:
Also, please try to avoid requiring readers to make mental context switches inside of a single line of code.  I.e. split out the `_substituteLocalizations` function call to at least one other line and a temporary variable, maybe more, depending on how readable it feels.

Reply:
Another way to win on readability here might be to add a 'getFirstLocalization' method in experimentUtils.ts.  Your call.